tests/test_init.py

```diff
@@ -83,54 +83,68 @@
-        "AUTOGRAPH_STAGE_FOCUS_USERNAME": "1",
-        "AUTOGRAPH_STAGE_FOCUS_V3_PASSWORD": "1",
-        "AUTOGRAPH_STAGE_FOCUS_V3_USERNAME": "1",
         "AUTOGRAPH_GPG_PASSWORD": "1",
         "AUTOGRAPH_GPG_USERNAME": "1",
         "AUTOGRAPH_LANGPACK_PASSWORD": "1",
         "AUTOGRAPH_LANGPACK_USERNAME": "1",
-        "AUTOGRAPH_MAR_PASSWORD": "1",
-        "AUTOGRAPH_MAR_USERNAME": "1",
-        "AUTOGRAPH_STAGE_MAR_PASSWORD": "1",
-        "AUTOGRAPH_STAGE_MAR_USERNAME": "1",
         "AUTOGRAPH_MAR_NIGHTLY_PASSWORD": "1",
         "AUTOGRAPH_MAR_NIGHTLY_USERNAME": "1",
+        "AUTOGRAPH_MAR_PASSWORD": "1",
         "AUTOGRAPH_MAR_RELEASE_PASSWORD": "1",
         "AUTOGRAPH_MAR_RELEASE_USERNAME": "1",
-        "AUTOGRAPH_MOZILLAVPN_PASSWORD": "1",
-        "AUTOGRAPH_MOZILLAVPN_USERNAME": "1",
+        "AUTOGRAPH_MAR_USERNAME": "1",
         "AUTOGRAPH_MOZILLAVPN_ADDONS_PASSWORD": "1",
         "AUTOGRAPH_MOZILLAVPN_ADDONS_USERNAME": "1",
         "AUTOGRAPH_MOZILLAVPN_DEBSIGN_PASSWORD": "1",
         "AUTOGRAPH_MOZILLAVPN_DEBSIGN_USERNAME": "1",
+        "AUTOGRAPH_MOZILLAVPN_PASSWORD": "1",
+        "AUTOGRAPH_MOZILLAVPN_USERNAME": "1",
         "AUTOGRAPH_OMNIJA_PASSWORD": "1",
         "AUTOGRAPH_OMNIJA_USERNAME": "1",
         "AUTOGRAPH_REFERENCE_BROWSER_PASSWORD": "1",
         "AUTOGRAPH_REFERENCE_BROWSER_USERNAME": "1",
-        "AUTOGRAPH_STAGE_REFERENCE_BROWSER_PASSWORD": "1",
+        "AUTOGRAPH_STAGE_AUTHENTICODE_SHA2_USERNAME": "1",
+        "AUTOGRAPH_STAGE_AUTHENTICODE_SHA2_PASSWORD": "1",
+        "AUTOGRAPH_STAGE_FENIX_PASSWORD": "1",
+        "AUTOGRAPH_STAGE_FENIX_USERNAME": "1",
+        "AUTOGRAPH_STAGE_FOCUS_PASSWORD": "1",
+        "AUTOGRAPH_STAGE_FOCUS_USERNAME": "1",
+        "AUTOGRAPH_STAGE_GPG_PASSWORD": "1",
+        "AUTOGRAPH_STAGE_GPG_USERNAME": "1",
+        "AUTOGRAPH_STAGE_LANGPACK_PASSWORD": "1",
+        "AUTOGRAPH_STAGE_LANGPACK_USERNAME": "1",
+        "AUTOGRAPH_STAGE_MAR_PASSWORD": "1",
+        "AUTOGRAPH_STAGE_MAR_USERNAME": "1",
+        "AUTOGRAPH_STAGE_MOZILLAVPN_ADDONS_PASSWORD": "1",
+        "AUTOGRAPH_STAGE_MOZILLAVPN_ADDONS_USERNAME": "1",
+        "AUTOGRAPH_STAGE_MOZILLAVPN_DEBSIGN_PASSWORD": "1",
+        "AUTOGRAPH_STAGE_MOZILLAVPN_DEBSIGN_USERNAME": "1",
+        "AUTOGRAPH_STAGE_OMNIJA_PASSWORD": "1",
+        "AUTOGRAPH_STAGE_OMNIJA_USERNAME": "1",
         "AUTOGRAPH_STAGE_REFERENCE_BROWSER_USERNAME": "1",
+        "AUTOGRAPH_STAGE_REFERENCE_BROWSER_PASSWORD": "1",
+        "AUTOGRAPH_STAGE_XPI_PASSWORD": "1",
+        "AUTOGRAPH_STAGE_XPI_USERNAME": "1",
+        "AUTOGRAPH_STAGE_XPI_PRIVILEGED_PASSWORD": "1",
+        "AUTOGRAPH_STAGE_XPI_PRIVILEGED_USERNAME": "1",
         "AUTOGRAPH_WIDEVINE_PASSWORD": "1",
         "AUTOGRAPH_WIDEVINE_USERNAME": "1",
+        "AUTOGRAPH_XPI_PASSWORD": "1",
         "AUTOGRAPH_XPI_PRIVILEGED_PASSWORD": "1",
         "AUTOGRAPH_XPI_PRIVILEGED_USERNAME": "1",
+        "AUTOGRAPH_XPI_USERNAME": "1",
```

Comment:
Keeping username and password next to each other might be better even if it means the list is not quite sorted?

Reply:
Fair!